src/projection/node/create-projection-node.ts

```diff
@@ -748,11 +748,24 @@ export function createProjectionNode<I>({
             return boxWithoutScroll
         }
 
-        applyTransform(box: Box): Box {
+        applyTransform(box: Box, transformOnly = false): Box {
             const withTransforms = createBox()
             copyBoxInto(withTransforms, box)
-            for (let i = 0; i < this.path.length; i++) {
+            for (let i = 1; i < this.path.length; i++) {
```

Comment:
Why do we skip the parent? Can we add a comment here to clarify?

Reply:
I think this was the root node but I've put it back to 0 as it doesn't really make a difference and is clearer